ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -19,6 +19,17 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_SS_CO2M_ZTC_STC_RTM:
+  description: "AC unit with heating and cooling (zone temp control)"
+  is_canonical: true
+  implements:
+  - FCU
+  - SS
+  - CO2M
+  - ZTC
+  - STC
```

Comment:
How can the unit control to both supply and zone temp?

Reply:
Apologies, this was also just supply temp monitoring not control.